l10n/en/products/vpn/shared.ftl

```diff
@@ -0,0 +1,38 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+### URL: https://www-dev.allizom.org/products/vpn/
+
+vpn-subscribe-link = Try { -brand-name-mozilla-vpn }
+vpn-waitlist-link = Join the Waitlist
+vpn-sign-in-link = Sign In
+vpn-available-countries = We currently offer { -brand-name-mozilla-vpn } in the US, the UK, Canada, New Zealand, Singapore and Malaysia.
+vpn-available-platforms = Available now: { -brand-name-windows } 10, { -brand-name-mac-short }, { -brand-name-android }, { -brand-name-ios } & { -brand-name-linux }
+vpn-availablity-long = The VPN is currently available for { -brand-name-windows } 10, { -brand-name-android } and { -brand-name-ios } devices in the US, the UK, Canada, New Zealand, Singapore and Malaysia.
+
+# Variables:
+#   $amount (string) - monthly subscription price using localized currency based on page locale / country e.g. US$4.99/month
+vpn-price-monthly = { $amount }/month
+
+vpn-features-encrypt = Device-level encryption
+
+# Variables:
+#   $servers (number) - number of available servers
+#   $countries (number) - number of available countries
+vpn-features-servers = { $servers }+ servers in { $countries }+ countries
```

Comment:
Do we need to localize the latest list of countries too? _(Writing out each country name, comma seperated?)_

Reply:
We have that too in the string `vpn-available-countries` 👍

Reply:
🙈